client/src/App.jsx

```diff
@@ -11,50 +14,21 @@ const withSuspense = (Component) => (
 
 const HomePage = React.lazy(() => import("./pages/HomePage"));
 
-// TODO: Add brand colors map
-// eslint-disable-next-line
-const BRAND_COLORS = {
-  BLUEBERRY: "#021944",
-  LYCHEE: "#F6A794",
-  MATCHA: "#B1D5B2",
-  MILK_TEA: "#F2DAC4",
-  TARO: "#E4CFF7",
-  THAI_TEA: "#E0A878",
-};
-
 const theme = createTheme({
   palette: {
-    background: { paper: BRAND_COLORS.MILK_TEA, default: "#FAF0E7" },
-    success: {
-      main: "#489878",
-    },
-    error: {
-      main: "#D21C1C",
-    },
-    info: {
-      main: "#005EA2",
-    },
-    warning: {
-      main: "#F9971E",
-    },
-    text: {
-      primary: "#2F3032",
-      secondary: "#C4C4C4",
-    },
+    ...color,
   },
-  fontsize: "16px",
   typography: {
-    color: "#2F3032",
-    fontFamily: "Poppins",
-    h1: { fontWeight: "bold", fontSize: "4.5rem" },
-    h2: { fontWeight: "bold", fontSize: "3.5rem" },
-    h3: { fontWeight: "bold", fontSize: "3rem" },
-    h4: { fontWeight: "bold", fontSize: "2rem" },
-    h5: { fontWeight: "bold", fontSize: "1.5rem" },
-    h6: { fontWeight: "bold", fontSize: "1rem" },
-    body1: { fontSize: "1.125rem" },
-    body2: { fontSize: "1.125rem", color: "#C4C4C4" },
-    title: { fontWeight: "500", fontSize: "4.5rem", lineHeight: "5.5rem" },
+    ...text,
+  },
+  components: {
+    MuiTypography: {
+      styleOverrides: {
+        root: {
+          fontFamily: "Poppins",
+        },
+      },
+    },
   },
 });
```

Comment:
Instead of abstracting just colors and text, it might be cleaner to just pull out the entire theme creation, either the object or the entire theme function itself. Splitting up the create theme and then objects that will most likely be only used in this one place creates two areas that devs have to look to see what is in `color` and `text` when these are all related and used in the same fashion

Reply:
yeah that makes sense I will change that